packages/kit/src/exports/vite/index.js

```diff
@@ -138,6 +140,11 @@ export async function sveltekit() {
 	return [...svelte(vite_plugin_svelte_options), ...kit({ svelte_config })];
 }
 
+/**
+ * If `true`, the server build has been completed and we're creating the client build
```

Comment:
why create a separate variable for this vs just looking to see whether `ssr` is set?

Reply:
because then we couldn't override the configured value, meaning that if someone had `build.ssr: true` in their config it would create an infinite loop